src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -1388,7 +1388,7 @@ internal static int FromHex(char digit)
                     : ((int)digit - (int)'a'))
                     + 10);
             }
-            throw new ArgumentException(nameof(digit));
+            throw new ArgumentException(SR.Argument_NotHexDigit, nameof(digit));
```

Comment:
Please do not change the exception type.  We must keep the same for app-compat for now.